src/Networking/IK.cpp

```diff
@@ -101,7 +104,7 @@ bool ParseIKPacket(json &message) {
     double cosElbowAngle = (crossSection*crossSection
       - ELBOW_LENGTH*ELBOW_LENGTH - SHOULDER_LENGTH*SHOULDER_LENGTH)
       /(-2*SHOULDER_LENGTH*ELBOW_LENGTH);
-    if (abs(cosElbowAngle) >= 1.0)
```

Comment:
I have another PR out that switches all the floating-point absolute values to `fabs` https://github.com/huskyroboticsteam/PY2020/pull/117

Do you have a strong preference which way we solve this (`fabs` vs `std::abs`)? Using `fabs` has the slight advantage that every time we read it it's a reminder that we need to be careful about this, but I don't have a terribly strong opinion.

Reply:
`std::abs` might be nice because it will do what we're expecting by writing `abs` and select the right type automatically; however you're right that the other way has the advantage of being more explicit. I don't really have a strong preference either, but if we choose one we probably ought to use it consistently throughout the codebase (and maybe add something to the style guide about it?)

Reply:
If it doesn't matter to you, then I lean towards the `fabs` solution, just because I already implemented it. I can add a style guide update to the other PR!